api/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM composer:2.0.8 AS composer
+FROM composer:2 AS composer
 
 WORKDIR /app
 
 # Install composer dependencies
 COPY composer.json .
 COPY composer.lock .
 
-RUN composer install --prefer-dist --no-interaction --no-scripts
+RUN composer install --prefer-dist --no-interaction --no-scripts --ignore-platform-req=php
```

Comment:
Can we add a comment here to remove `--ignore-platform-req=php` when we're on 8?